### DISCUSSION TITLE
loosing a lot of fps in game

### DISCUSSION THREAD

**Discussion by Morcooooooo** [ppy/osu#16614] [Q&A]
when I'm in the main menu, I easily reach high perf but once I start a map I lose about 2 thirds of my update fps, half of draw and many for audio and input (still over 1000 for these two). So my game is no longer very fluid and my cursor has latency
I'm on 2022.118.0

screens : 
(main) 
![main](https://user-images.githubusercontent.com/72506069/151016867-0fbd4725-38c5-41c0-9f40-5e8d2379c293.jpg)

(in game) 
![ingame](https://user-images.githubusercontent.com/72506069/151016880-9f250ef9-961d-4d8b-b870-010fca116f37.jpg)

logs : 
[database.log](https://github.com/ppy/osu/files/7935337/database.log)
[network.log](https://github.com/ppy/osu/files/7935338/network.log)
[performance.log](https://github.com/ppy/osu/files/7935339/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/7935340/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7935341/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/7935342/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/7935343/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7935344/updater.log)

**Comment by bdach** [comment]
your cursor should not have latency if the input framerate is over 1000. also what is your screen's refresh rate? by the limit of 480Hz i'm guessing that it's either 60Hz and you're running on 8x frame limiter or it's 120Hz and you're running 4x. would recommend trying to take both down a notch to 4x and 2x respectively and seeing if it helps.

in general a video of what you feel looks wrong would come a long way, with the frame graph showing (press ctrl-f11 again when standard fps counter is showing). a textual description is insufficient to debug this.

**Reply by LumpBloom7** [reply]
At least in my experience on windows, playing in borderless fullscreen does incur a hefty amount of input lag regardless  of what the numbers show.

Maybe OP should see if they are playing on borderless? And if so, whether exclusive fullscreen helps.

**Reply by Morcooooooo** [reply]
I have a 120Hz with 4x, and I tried to use 2x and it changed nothing about it, neither the fps, nor the latency of the slider (which cannot be shown on the video)
When i made the video for the problem, i saw that the more objects on the screen, the more big loss i had. And there are also some lag spikes 
(I will put the link of the video later because I have a very bad connection and it takes a long time to upload)

**Reply by Morcooooooo** [reply]
there https://streamable.com/ig760w

**Comment by peppy** [comment]
Nothing shown in your screenshots will affect performance. You are likely not running exclusive fullscreen or experiencing placebo.

**Reply by Morcooooooo** [reply]
I'm running fullscreen, and I tested well on lazer and on stable, and I notice a much higher latency on lazer
Also if it helps i have two cpu's in my pc and i was told before that could cause problems with osu!

**Reply by Morcooooooo** [reply]
and there, a map with more objects on screen https://streamable.com/o80zf6

**Comment by peppy** [comment]
Locking this for now. Please use stable for the time being if it works better for you. It's likely an issue with exclusive fullscreen or something else and doesn't need further discussion.
